beacon-chain/state/stategen/getter.go

```diff
@@ -65,7 +65,7 @@ func (s *State) StateByRoot(ctx context.Context, blockRoot [32]byte) (state.Beac
 	return s.loadStateByRoot(ctx, blockRoot)
 }
 
-// BalancesByRoot retrieves the effective balances of all validators at the
+// BalancesByRoot retrieves the effective balances of all active validators at the
```

Comment:
Can you please update the function name and the comments to indicate these is non-slashed balances? 
While it's ok now, I'm afraid someone tomorrow decide to use this function for consensus-related functions and other clients still account for slashed balances